jdaviz/utils.py

```diff
@@ -480,6 +483,17 @@ def get_subset_type(subset):
                 # subset type:
                 continue
 
+            # check for spectral coordinate in GWCS by looking for SpectralFrame
+            if isinstance(ss_data.coords, gwcs):
+                if isinstance(ss_data.coords, (SpectralFrame, SpectralGWCS)):
+                    return 'spectral'
+                elif isinstance(ss_data.coords, CompositeFrame):
+                    if np.any([isinstance(frame, SpectralFrame) for frame in
+                               ss_data.coords.output_frame.frames]):
+                        return 'spectral'
+                else:
+                    continue
```

Comment:
doesn't seem possible to get here because of the previous `else: continue`

Reply:
Perhaps but eh, better safe than sorry. I don't remember if I had a specific case that escaped the other handling.

Reply:
It's been long enough that it makes me nervous to remove it 😆

Reply:
but its literally impossible for the code to reach here, right?  If we need to handle this case, should it be somewhere else?